jekyll/_cci2/circleci-images.md

```diff
@@ -213,14 +217,46 @@ if you want
 to add browsers to the `circleci/golang:1.9` image,
 use the `circleci/golang:1.9-browsers` image.
 
+### Next-Gen Language Images
+{:.no_toc}
+
+Like the legacy images, the next-gen language images are convenience images for common programming languages.
+These images include both the same relevant language and [commonly-used tools](#pre-installed-tools).
+A language image should be listed first under the `docker` key in your configuration,
+making it the [primary container]({{ site.baseurl }}/2.0/glossary/#primary-container){:target="_blank"} during execution.
+
+CircleCI is developing next-gen images for the languages below.
+
+- Android
+- Clojure
+- [Elixir](https://circleci.com/developer/images/image/cimg/elixir)
+- [Go (Golang)](https://circleci.com/developer/images/image/cimg/go)
+- [Node.js](https://circleci.com/developer/images/image/cimg/node)
+- [OpenJDK (Java)](https://circleci.com/developer/images/image/cimg/openjdk)
+- [PHP](https://circleci.com/developer/images/image/cimg/php)
+- [Python](https://circleci.com/developer/images/image/cimg/python)
+- [Ruby](https://circleci.com/developer/images/image/cimg/ruby)
+- [Rust](https://circleci.com/developer/images/image/cimg/rust)
+
+If your language is not listed,
+CircleCI also maintains a [Dockerfile Wizard](https://github.com/circleci-public/dockerfile-wizard)
```

Comment:
This tool isn't maintained. Might be best to remove.

We can also reference CircleCI Ideas. If a user wants to request an image that we don't have they should:

1. Check to see if that "idea" is already on CircleCI Ideas. If it is, upvote it.
2. If not, create it and set the category as "images".
3. They can optionally go and market their "idea" to friends, co-workers, forums, and Slack communities in order to help it build traction.

If we see an idea take off, we're consider building it officially.